app/src/main/java/org/kiwix/kiwixmobile/zim_manager/fileselect_view/ZimFileSelectFragment.java

```diff
@@ -216,19 +221,19 @@ public boolean onActionItemClicked(ActionMode mode, MenuItem item) {
             selectedFileShareIntent.putParcelableArrayListExtra(Intent.EXTRA_STREAM,
                 selectedFileContentURIs); // Intent Extra for storing the array list of selected file content URIs
 
-            if (selectedFileContentURIs
-                != null) {  // Grant temporary access permission to the intent receiver for the content URIs
-              selectedFileShareIntent.setFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION);
-            }
+            // Grant temporary access permission to the intent receiver for the content URIs
+            selectedFileShareIntent.setFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION);
 
-            /**
+            /*
              * Since a different app may be used for sharing everytime (E-mail, Cloud Upload, Wifi Sharing, etc.),
              * so force an app chooser dialog every time some selected files are to be shared.
              */
+
             Intent shareChooserIntent = Intent.createChooser(selectedFileShareIntent,
                 getResources().getString(R.string.selected_file_cab_app_chooser_title));
 
-            if (shareChooserIntent.resolveActivity(getActivity().getPackageManager()) != null) {
+            if (shareChooserIntent.resolveActivity(
+                getActivity().getPackageManager()) != null) {
```

Comment:
This line must be change to `if (shareChooserIntent.resolveActivity(zimManageActivity.getPackageManager()) != null) {` to remove the lint warning as I have told you before and remove all the lint warnings in this file.

Reply:
@abdulwd Done.Thankyou for assisting me.